barretenberg/cpp/scripts/test_civc_standalone_vks_havent_changed.sh

```diff
@@ -13,7 +13,7 @@ cd ..
 # - Generate a hash for versioning: sha256sum bb-civc-inputs.tar.gz
 # - Upload the compressed results: aws s3 cp bb-civc-inputs.tar.gz s3://aztec-ci-artifacts/protocol/bb-civc-inputs-[hash(0:8)].tar.gz
 # Note: In case of the "Test suite failed to run ... Unexpected token 'with' " error, need to run: docker pull aztecprotocol/build:3.0
-pinned_short_hash="bd98634a"
+pinned_short_hash="3087efc4"
```

Comment:
Is it clear what caused this change? Maybe just a reordering of certain constraints